pkg/server/handler_test.go

```diff
@@ -15,36 +15,84 @@ import (
 
 	"github.com/gorilla/mux"
 	"github.com/gruntwork-io/terratest/modules/terraform"
+
 	localkms "github.com/nimbolus/terraform-backend/pkg/kms/local"
 	locallock "github.com/nimbolus/terraform-backend/pkg/lock/local"
 	"github.com/nimbolus/terraform-backend/pkg/storage/filesystem"
+	tf "github.com/nimbolus/terraform-backend/pkg/terraform"
 )
 
-var terraformBinary = flag.String("tf", "terraform", "terraform binary")
-
-func TestServerHandler(t *testing.T) {
-	s := httptest.NewServer(NewStateHandler())
-	defer s.Close()
-
-	address, err := url.JoinPath(s.URL, "/state/project1/example")
+func NewStateHandler() http.Handler {
+	store, err := filesystem.NewFileSystemStorage(filepath.Join("./handler_test", "storage"))
 	if err != nil {
-		t.Fatal(err)
+		panic(err)
```

Comment:
why panic?

Reply:
Because the helper function did not have a `testing.T` before. But I agree and changed it.